.travis.yml

```diff
@@ -44,13 +44,13 @@ before_install:
 
 install:
   - pip install -U setuptools pip
-  - pip install unittest2 mock pycurl flake8 tox-travis coveralls
+  - pip install pycurl flake8 coveralls
```

Comment:
`tox-travis` was useless as we don't use tox to run tests on Travis.

`pytest` and any additional test modules will be installed when running `python setup.py install`, so no need to install them manually before.

Reply:
Nice to see this cleaned out.